### PR TITLE
DevOps Pipeline set up

### PR DESCRIPTION
**Pull request by satya-sanikommu**

### CODE REVIEW

File: src/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.csproj [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="SFA.DAS.Funding.ApprenticeshipEarnings.Types" Version="0.1.5-prerelease-90" />
     <PackageReference Include="SFA.DAS.NServiceBus" Version="17.0.13" />
     <PackageReference Include="SpecFlow" Version="3.9.40" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="6.0.1" />
```
Comment (tom-gough) [inline src/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.csproj:15]: Is this needed?

File: src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/FundingPeriodAssertions.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ public class FundingPeriodAssertions : GenericCollectionAssertions<FundingPeriod
         {
             Execute.Assertion
                 .Given(() => subjArray[i])
-                .ForCondition(v => v.AgreedPrice*0.80m == targetValue)
+                .ForCondition(v => v.AgreedPrice == targetValue)
```
Comment (tom-gough) [inline src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/FundingPeriodAssertions.cs:22]: logic change not devops code, is this intended to be in this PR?

Comment (satya-sanikommu) [reply src/SFA.DAS.Funding.SystemAcceptanceTests/Helpers/FundingPeriodAssertions.cs:22]: @tom-gough this is due to the problem with the commit history re-writing. This is supposed to be in main already

File: src/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.csproj (hunk 1 above)
Comment (ruslanss) [reply src/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure/SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.csproj:15]: yes this resolves package scan issues

**Review by ruslanss — APPROVED** [review]

**Review by HussainsGitHub — APPROVED** [review]

**Pull request merged**